RELEASE.md

```diff
@@ -5,22 +5,22 @@ Steps to release a new version of the Jaeger Operator:
 
 1. Change the `versions.txt `so that it lists the target version of the Jaeger (if it is required). Don't touch the operator version it will be changed automatically in the next step.
 
-1. Run `make prepare-release OPERATOR_VERSION=1.27.0`, using the operator version that will be released.
```

Comment:
I guess we don't have to update versions in this file

Reply:
We don't have it